### ISSUE TITLE
"STS\ZipStream\Exceptions\FilenameMissingException" Unable to add file but file exists

### ISSUE THREAD

**Issue by XanCeegor** [stechstudio/laravel-zipstream#16]
```
$zip_file = Zip::create("myfile.zip");
$zip_file->add(storage_path('app/files/qPygS13dEaa8KTDnzvx69fVBwqXiSJl6YX06NvcJ.pdf'));
return $zip_file;
```

Hi. I'm using the above code to try and ZIP a PDF file on my local Laravel machine but I get the exception in the title. The file definitely exists. This happens for all files I try to add. 
When I use `return response()->download(storage_path('app/files/qPygS13dEaa8KTDnzvx69fVBwqXiSJl6YX06NvcJ.pdf'));` as a test it returns the file correctly.

Is there something that I'm doing wrong? My dev enviroment is a Win10 machine with XAMPP running for dev purposes.
Any help would be great! 
Thanks.

**Comment by jszobody** [comment]
Can you run this code and tell me what you get:

```php
$path = storage_path('app/files/qPygS13dEaa8KTDnzvx69fVBwqXiSJl6YX06NvcJ.pdf');
dd($path, file_exists($path));
```

I'd like to see the absolute path that you're getting and to verify that PHP can see the file exists.

**Comment by jszobody** [comment]
Upgrade to v2.1 and try again.

This package assumed that local filepaths start with a forward slash, and that's not going to be the case on Windows. I believe I just resolved it in v2.1. I don't have a Windows dev machine to test on, let me know if it's still giving you issues.

**Comment by XanCeegor** [comment]
Hi. 
This did indeed solve my problem.
Thank you for this package! It's way faster than the other package I was using and the fact that it doesn't store the archive file on the server first is awesome!

Happy new year by the way!
